### ISSUE TITLE
Lua on blind-distance

### ISSUE THREAD

**Issue by Wubsy** [codecombat/codecombat#4626]
The code is set up for javascript, despite me setting the language to Lua. It should be

```lua
-- Tell the wizard the distance to the coming ogres.

-- This function finds the nearest enemy and returns the distance to it.
function nearestEnemyDistance() 
    local enemy = hero:findNearestEnemy()
    -- If there is no enemy, the function returns 0.
    local result = 0
    if enemy then 
      local  result = hero:distanceTo(enemy)

    return result
    end
end

while true do
-- Call nearestEnemyDistance() and
    -- save the result in the variable enemyDistance.
    local enemyDistance = nearestEnemyDistance()
    -- If the enemyDistance is greater than 0: 
    
        -- Say the value of enemyDistance variable.
        
end

```

**Comment by nwinter** [comment]
@Wubsy As Lua and CoffeeScript are somewhat experimental languages for us, we don't have sample code for all levels in those languages. I'd love your help in adding sample code to any levels where you don't see Lua show up if you're willing to help? Here's a quick video I made showing how to do it:

https://www.dropbox.com/s/arq5e6ng5az2top/Adding%20Lua%20sample%20code%20in%20the%20level%20editor.mp4?dl=0

Basically, you open https://codecombat.com/editor/level/blind-distance, double-click on the hero placeholder, find the Programmable Component, find the `plans` entry, open `languages`, and add/edit the sample code for the target language (here Lua) by copying from the Python and making the appropriate changes. When done, you can submit a patch by clicking the save icon in the upper right of the level editor.

**Comment by Bryukh** [comment]
@Wubsy thanks! I've merged the lua patch for that level. Usually, I process patches on Mondays. But feel free to poke me in Slack (use the mention @bryukh) to process them early.
